react-graphql-apollo-tutorial.md

```diff
@@ -1570,32 +1564,32 @@ const RepositoryItem = ({ ... }) => (
+An addtional benefit of the optimistic response is that it makes the count of watchers updates optimistic, too. The function used in the `update` prop is called twice now, the first time with the optimistic response, and the second with a response from GitHub's GraphQL API. It makes sense to capture identical information in the optimistic response expected as a mutation result in the function passed to the `update` prop of the Mutation component. For instance, if you don't pass the `id` property in the `optimisticResponse` object, the function passed to the `update` prop throws an error, because it can't retrieve the repository from the cache without an identifier.
 
-At this point in time, it becomes debatable whether the everything in the Mutation component becomes too verbose. Using the Render Props pattern co-locates the data-layer even more to the view-layer than Higher-Order Components. One could say it doesn't co-locate the data-layer but inserts it into the view-layer. When all these optimizations, such as the `update` and `optimisticResponse` props, are put into the Render Prop Component, it becomes often too verbose. You have to find your own strategy as individual or team to keep things concise for these circumstances. Personally I see four different ways to solve this issue:
+At this point, it becomes debatable whether or not the Mutation component becomes too verbose. Using the Render Props pattern co-locates the data layer even more to the view-layer than Higher-Order Components. One could argue it doesn't co-locate the data-layer, but inserts it into the view-layer. When optimizations like the `update` and `optimisticResponse` props are put into the Render Prop Component, it can become to verbose for a scaling application. I advise using techniqes you've learned as well as your own strategies to keep your source concise. I see four different ways to solve this issue:
 
-* (1) keeping everything in the Mutation component inlined (e.g. `optimisticResponse`)
-* (2) extracting everything as variables (e.g. `update`)
-* (3) doing a combination of (2) and (3) whereas only the most verbose parts are extracted
-* (4) using Higher-Order Components instead of Render Props to co-locate data-layer rather than inserting it in the view-layer
+* Keep the Mutation component inlined (e.g. `optimisticResponse`)
+* Extracting everything as variables (e.g. `update`)
```

Comment:
Maybe "Extracting the inlined declarations as variable (see update)"

Reply:
These aren't editing corrections, but rewrites for content you decided to change.  They should be handled after the merge.